articles/aks/learn/tutorial-kubernetes-workload-identity-IdWeb.md

```diff
@@ -0,0 +1,369 @@
+---
+title: Tutorial - Use Microsoft.Identity.Web library to leverage workload identity with an application on Azure Kubernetes Service (AKS)
+description: In this Azure Kubernetes Service (AKS) tutorial, you deploy an Azure Kubernetes Service cluster and configure an application to use a workload identity with Microsoft.Identity.Web.
+ms.topic: tutorial
+ms.custom: devx-track-azurecli
+ms.date: 05/31/2023
+---
+
+# Tutorial: Use Microsoft.Identity.Web library to leverage workload identity with an application on Azure Kubernetes Service (AKS)
+
+Azure Kubernetes Service (AKS) is a managed Kubernetes service that lets you quickly deploy and manage Kubernetes clusters. In this tutorial, will:
+
+* Deploy an AKS cluster using the Azure CLI with OpenID Connect (OIDC) Issuer and managed identity.
+* Create an Azure Key Vault and secret.
+* Create an Azure Active Directory (Azure AD) workload identity and Kubernetes service account.
+* Configure the managed identity for token federation.
+* Deploy the workload and verify authentication with the workload identity.
+
+## Before you begin
+
+* This tutorial assumes a basic understanding of Kubernetes concepts. For more information, see [Kubernetes core concepts for Azure Kubernetes Service (AKS)][kubernetes-concepts].
```

Comment:
@RojaEnnam : I'm not familiar with the syntax `[title][something]`, did you mean to have `[title](url)`?